internal/host/transition.go

```diff
@@ -511,6 +511,9 @@ func (th *transitionHandler) PostRefreshHost(reason string) stateswitch.PostTran
 		if err != nil {
 			return err
 		}
+		if sHost.host.Progress.CurrentStage == models.HostStageWritingImageToDisk {
+			template = statusInfoInstallationInProgressWritingImageToDiskTimedOut
+		}
```

Comment:
Do we want to set this here rather than passing a different "reason" argument?

Reply:
I thought about it also but we pass it in https://github.com/openshift/assisted-service/blob/master/internal/host/statemachine.go#L325 and I don't know the host stage there.
Since we manipulate the status info in this method anyway - I thought it is better.